src/main/java/com/autotune/analyzer/services/ListRecommendations.java

```diff
@@ -209,7 +211,6 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
                 List<ListRecommendationsAPIObject> recommendationList = new ArrayList<>();
                 for (KruizeObject ko : kruizeObjectList) {
                     try {
-                        LOGGER.debug(ko.getKubernetes_objects().toString());
```

Comment:
Why is this getting removed?

Reply:
This generates a large log, which I felt was unnecessary at times since we already have sufficient pre- and post-trial logs for debugging. The excessive logging clutters the output, so I removed it to simplify the debugging process.